exporter/Makefile

```diff
@@ -0,0 +1 @@
+include ../Makefile.Common
```

Comment:
Why is this file necessary?

Reply:
It is not, it was leftover from the testing of having `exporter` as its own module